Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityUtilities.cs

```diff
@@ -12,7 +15,34 @@ namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input
     public static class WindowsMixedRealityUtilities
     {
 #if WINDOWS_UWP
+#if ENABLE_DOTNET
+        [DllImport("DotNetNativeWorkaround.dll", EntryPoint = "MarshalIInspectable")]
+        private static extern void GetSpatialCoordinateSystem(IntPtr nativePtr, out SpatialCoordinateSystem coordinateSystem);
+
+        public static SpatialCoordinateSystem SpatialCoordinateSystem => spatialCoordinateSystem ?? (spatialCoordinateSystem = GetSpatialCoordinateSystem(WorldManager.GetNativeISpatialCoordinateSystemPtr()));
+
+        /// <summary>
+        /// Helps marshal WinRT IInspectable objects that have been passed to managed code as an IntPtr.
+        /// On .NET Native, IInspectable pointers cannot be marshaled from native to managed code using Marshal.GetObjectForIUnknown.
+        /// This class calls into a native method that specifically marshals the type as a specific WinRT interface, which
+        /// is supported by the marshaller on both .NET Core and .NET Native.
+        /// </summary>    
```

Comment:
Nit: In the remarks, it might be worth adding a link to the docs that you're adding as well. I know that this might be a circular reference, but it could help people understand "hey, I just hit this LogError, what do I do?"

Reply:
Oh, since this is a private function, the summary / remarks don't actually get published anywhere :/

Reply:
That's okay - my comments were more about fulfilling the needs of someone who is reading this code directly (i.e. they ask, why don't hands work?) or they see the log message - they might then come directly to this code, so having the link here would reduce the friction for figuring out how this part works.

Reply:
Cool yeah, I can put a more full link in. I was thinking more getting an xref working, which isn't super translatable while reading to figure out exactly where you need to go.

Reply:
Added